modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImpl.java

```diff
@@ -197,6 +198,12 @@ public class ServiceRegistryJpaImpl implements ServiceRegistry, ManagedService {
   /** Configuration key for the retrieval of service statistics: Do not consider jobs older than max_job_age (in days) */
   protected static final String OPT_SERVICE_STATISTICS_MAX_JOB_AGE = "org.opencastproject.statistics.services.max_job_age";
 
+  /** Configuration key for the encoding preferred worker nodes */
+  protected static final String OPT_ENCODINGWORKERS = "org.opencastproject.encoding.workers";
+
+  /** Configuration key for the encoding workers load threshold */
+  protected static final String OPT_ENCODINGTHRESHOLD = "org.opencastproject.encoding.workers.threshold";
+
```

Comment:
```suggestion
  /** Configuration key for the encoding preferred worker nodes */
  protected static final String OPT_ENCODING_WORKERS = "org.opencastproject.encoding.workers";

  /** Configuration key for the encoding workers load threshold */
  protected static final String OPT_ENCODING_THRESHOLD = "org.opencastproject.encoding.workers.threshold";

```
Using snake case would be easier to read.

Reply:
Thank you. I applied these changes.

Reply:
Thx!